js/app/nav.js

```diff
@@ -78,6 +79,7 @@ function cloneSecondaryNav() {
                .removeClass('primary-nav__link col')
                .addClass('secondary-nav__link')
                .appendTo('.js-nav-clone__list li:nth-child(' + index + ')');
+            $(this).parent().removeClass('hidden');
```

Comment:
Likewise, we can use the jQuery show function here too https://api.jquery.com/show/